[Improve]Remove http v1 code and disable mini load by default

hf200012 opened the pull request:
1. http v2 has been actually tested in production, and it is completely replaceable to have http code. In order to simplify code maintenance, remove the previous http part of the code
2. The mini load function can completely pass the stream load function. In the next version, the mini load transition is disabled through the switch, and then this code is removed in the subsequent version. If users need to use the mini load function, they can add  in fe.conf `disable_mini_load=false` to enable, the default is disabled


# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes)
2. Has unit tests been added: (No Need)
4. Has document been added or modified: (Yes)
5. Does it need to update dependencies: (No)
6. Are there any changes that cannot be rolled back: (No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

hf200012 committed:
Remove http v1 code and disable mini load by default

Remove http v1 code and disable mini load by default

hf200012 committed:
1

1

yiguolei commented on fe/fe-core/src/main/java/org/apache/doris/PaloFe.java:
```diff
@@ -134,24 +134,13 @@ public static void start(String dorisHomeDir, String pidDir, String[] args) {
-            }
+            HttpServer httpServer2 = new HttpServer();
+            httpServer2.setPort(Config.http_port);
+            httpServer2.setMaxHttpPostSize(Config.jetty_server_max_http_post_size);
```
httpServer is ok

yiguolei commented on fe/fe-core/src/test/java/org/apache/doris/http/HttpAuthManagerTest.java:
```diff
@@ -31,15 +33,19 @@ public void testNormal() {
+
         String noExistSession = "no-exist-session-id";
-        Assert.assertNull(authMgr.getSessionValue(noExistSession));
+        sessionIds.clear ();
```
clear()

yiguolei commented on fe/fe-core/src/main/java/org/apache/doris/journal/bdbje/BDBDebugger.java:
```diff
@@ -85,21 +85,14 @@ public void startDebugMode(String dorisHomeDir) {
+        HttpServer httpServer = new HttpServer();
+        httpServer.setPort(Config.http_port);
+        httpServer.setMaxHttpPostSize(Config.jetty_server_max_http_post_size);
+        httpServer.setAcceptors(Config.jetty_server_acceptors);
```
Why set these parameters but not set them in the past?

yiguolei commented on fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/LoadAction.java:
```diff
@@ -59,9 +62,14 @@
-        this.isStreamLoad = false;
-        executeCheckPassword(request, response);
-        return executeWithoutPassword(request, response, db, table);
+        if(!Config.disable_mini_load) {
```
it is better to use if (condition) xxx not use if (!condition)

yiguolei commented on fe/fe-core/src/main/java/org/apache/doris/common/Config.java:
```diff
@@ -1650,4 +1634,10 @@
+    /**
+     * Mini load operation is disabled by default
+     */
+    public static boolean disable_mini_load = true;
```
Missing @ConfField?

hf200012 committed:
2

2

hf200012 committed:
Add jetty container minimum and maximum number of threads can be configured

Add jetty container minimum and maximum number of threads can be configured, and modify the default minimum and maximum values: 20,400

yiguolei reviewed (dismissed):
LGTM

hf200012 committed:
Merge branch 'master' into remove-mini-load

morningman reviewed (changes requested):
Please don't put multiple types of changes into one PR.
Please split this PR into 3 parts:
1. disable mini load
2. remove http v1
3. modify http thread config
